webrender/src/device/gfx/descriptor.rs

```diff
@@ -435,54 +405,143 @@ where
         unsafe { device.write_descriptor_sets(descriptor_writes) };
         new_set
     }
+}
+
+struct BoundDecsriptor<B: hal::Backend> {
+    descriptor: DescriptorSet<B>,
+    bound: bool,
+}
+
+struct DecsriptorBundle<B: hal::Backend> {
```

Comment:
Not sure why we are having this descriptor re-usal infrastructure in this PR.
As we discussed on the call, this is supposed to be a single descriptor with dynamic offset (for the uniform buffer that carries out the projection matrix), which is just re-used constantly for many passes.